cloud_formation/configs/activities.py

```diff
@@ -169,6 +184,12 @@ def update(session, domain):
     resp = input('Replace step functions: [Y/n]:')
     if len(resp) == 0 or (len(resp) > 0 and resp[0] in ('Y', 'y')):
```

Comment:
Could use this from the lib.utils function library:
```python
if utils.get_user_confirm("Replace step functions:"):
         delete_sfns(session, domain)
         # Need to delay so AWS actually removes the step functions before trying to create them
        delay = 30
        print("Step Functions deleted, waiting for {} seconds".format(delay))
        time.sleep(delay)
        post_init(session, domain)```

Reply:
It could be. There are several locations where `get_user_confirm` can be used. I'll put this on the list of updates to do during the refactor.